oc-chef-pedant/lib/pedant/rspec/matchers.rb

```diff
@@ -95,12 +83,38 @@ def array_matches?(spec, value)
         # Note this is going to be slow for large arrays,
         # though our actual usage shows we don't have any arrays on a
         # scale that would matter.
-        return true if spec.all? { |s| value.include? s }
-        # Our comparisons are more complex than simple value
-        spec.each_with_index  do |newspec, x|
-          return false unless PedantHashComparator.new(newspec, @mode).matches? value[x]
+        return true if spec.all? do |s|
```

Comment:
[nit] isn't this `return true if ...` followed by `return false` equivalent to ending the method with

    spec.all? do ...

? 😉